### DISCUSSION TITLE
Google Classroom

### DISCUSSION THREAD

**Discussion by ch5rus** [catppuccin/catppuccin#2400] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

similar to gmail i guess? surely theyre similar

### Any additional comments?

i could make it myself if my acoustic beginner css pulls through

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by isabelroses** [comment]
There is a open PR for this, please see https://github.com/catppuccin/userstyles/pull/755

**Reply by ch5rus** [reply]
> There is a open PR for this, please see https://github.com/catppuccin/userstyles/pull/755

Legend thanks
